pydra/engine/core.py

```diff
@@ -733,22 +742,23 @@ def __getattr__(self, name):
             return self.name2obj[name]
         return self.__getattribute__(name)
 
-    @property
-    def done_all_tasks(self):
-        """
-        Check if all tasks from the graph are done.
-
-        .. important ::
-            The fact that all tasks are reported as done
-            doesn't mean that results of the workflow
-            are available.
-            That can be checked with :py:meth:`~Workflow.done`.
-
-        """
-        for task in self.graph.nodes:
-            if not task.done:
-                return False
-        return True
+    # TODO: not sure if we will need this, if not, should be removed
```

Comment:
remove if not needed.

Reply:
I will, just not completely sure if everything will work, but looks like